yarn.lock

```diff
@@ -656,9 +656,9 @@
     adal-node "^0.1.28"
 
 "@babel/cli@^7.10.1":
-  version "7.10.3"
-  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.10.3.tgz#4ea83bd997d2a41c78d07263ada3ec466fb3764b"
-  integrity sha512-lWB3yH5/fWY8pi2Kj5/fA+17guJ9feSBw5DNjTju3/nRi9sXnl1JPh7aKQOSvdNbiDbkzzoGYtsr46M8dGmXDQ==
+  version "7.10.4"
```

Comment:
The large diff here tells me you ran `yarn` in a folder other than monorepo root. That messes things up unfortunately. I'd recommend adding dependencies manually.
Re-run yarn from root to fix this lock file :)